[2025-02-14 02:27:26,163] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: Project.task_zero 2025-02-14T02:27:25.413253+00:00 [queued]>
[2025-02-14 02:27:26,187] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: Project.task_zero 2025-02-14T02:27:25.413253+00:00 [queued]>
[2025-02-14 02:27:26,188] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2025-02-14 02:27:26,188] {taskinstance.py:1088} INFO - Starting attempt 1 of 3
[2025-02-14 02:27:26,189] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2025-02-14 02:27:26,199] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_zero> on 2025-02-14T02:27:25.413253+00:00
[2025-02-14 02:27:26,209] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Project', 'task_zero', '2025-02-14T02:27:25.413253+00:00', '--job-id', '114', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmprxuj11ko', '--error-file', '/tmp/tmp30qw8y_s']
[2025-02-14 02:27:26,204] {standard_task_runner.py:52} INFO - Started process 922 to run task
[2025-02-14 02:27:26,210] {standard_task_runner.py:77} INFO - Job 114: Subtask task_zero
[2025-02-14 02:27:26,266] {logging_mixin.py:104} INFO - Running <TaskInstance: Project.task_zero 2025-02-14T02:27:25.413253+00:00 [running]> on host ad36e95eb2b4
[2025-02-14 02:27:26,335] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Project
AIRFLOW_CTX_TASK_ID=task_zero
AIRFLOW_CTX_EXECUTION_DATE=2025-02-14T02:27:25.413253+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-14T02:27:25.413253+00:00
[2025-02-14 02:27:26,340] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2025-02-14 02:27:26,343] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'pip install beautifulsoup4']
[2025-02-14 02:27:26,356] {subprocess.py:75} INFO - Output:
[2025-02-14 02:27:27,697] {subprocess.py:79} INFO - Defaulting to user installation because normal site-packages is not writeable
[2025-02-14 02:27:27,808] {subprocess.py:79} INFO - Requirement already satisfied: beautifulsoup4 in /home/***/.local/lib/python3.6/site-packages (4.12.3)
[2025-02-14 02:27:27,818] {subprocess.py:79} INFO - Requirement already satisfied: soupsieve>1.2 in /home/***/.local/lib/python3.6/site-packages (from beautifulsoup4) (2.3.2.post1)
[2025-02-14 02:27:32,268] {subprocess.py:83} INFO - Command exited with return code 0
[2025-02-14 02:27:32,301] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=Project, task_id=task_zero, execution_date=20250214T022725, start_date=20250214T022726, end_date=20250214T022732
[2025-02-14 02:27:32,331] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-14 02:27:32,343] {local_task_job.py:151} INFO - Task exited with return code 0
